pkg/kepctl/kepctl.go

```diff
@@ -481,3 +482,25 @@ func (c *Client) PrintTable(configs []PrintConfig, proposals []*keps.Proposal) {
 	}
 	table.Render()
 }
+
+// PrintYAML outputs keps array as YAML to c.Out
+func (c *Client) PrintYAML(proposals []*keps.Proposal) {
+	data, err := yaml.Marshal(proposals)
```

Comment:
nit: Could you write one function with an additional argument to flag the output type?

Reply:
👋🏽, do you mean something like the following?
```
func (c *Client) Print(proposals []*keps.Proposal, output string) {
    switch output:
    case "table":
		// logic for table output type
	case "yaml":
		// logic for yaml  output type
	case "json":
		// logic for json  output type
	}
}
```

If that is the case, this is already handled a level up in the logic path, and these methods handle the printing logic for that specific output type.

Reply:
Hi. This is a nit/question. I saw the switch statement.
Just wondering if one function could handle all output types, but I see that the table function is bit different.

Reply:
> Just wondering if one function could handle all output types, but I see that the table function is bit different.

Yeah. The printer logic is a bit different for the `table` output type compared to `json` or `yaml`. That was the reason I kept the functions a bit different. A tad bit of inspiration was also taken from the printer flow of `kubectl`.